public/app/features/alerting/unified/components/rules/central-state-history/CentralAlertHistoryScene.tsx

```diff
@@ -180,7 +287,7 @@ export const FilterInfo = () => {
             </Trans>
             <pre>{`{severity="critical", instance=~"cluster-us-.+"}`}</pre>
             <Trans i18nKey="alerting.central-alert-history.filter.info.label2">Invalid use of spaces:</Trans>
-            <pre>{`{severity= "alerting.critical"}`}</pre>
+            <pre>{`{severity= "critical"}`}</pre>
```

Comment:
nit: I think this one is now a duplicate? :)

Reply:
Oof I totally missed that, still feels redundant and I think with the new utf8 parser it might accept the extra space as part of the label key or value.

Reply:
exactly, it's different use case